src/fundus/publishers/de/sz.py

```diff
@@ -14,9 +15,14 @@
 
 class SZParser(ParserProxy):
     class V1(BaseParser):
-        _paragraph_selector = CSSSelector('main [itemprop="articleBody"] > p, ' "main .css-korpch > div > ul > li")
+        _paragraph_selector = XPath(
+            "//div[@itemprop='articleBody']//p[@data-manual='paragraph'" " and not(contains(text(), '© dpa-infocom'))]"
+        )
         _summary_selector = CSSSelector("main [data-manual='teaserText']")
-        _subheadline_selector = CSSSelector("main [itemprop='articleBody'] > h3")
+        _subheadline_selector = XPath(
+            "//div[@itemprop='articleBody']//h3[@data-manual='subheadline']|"
+            "//div[@itemprop='articleBody']//h2[@data-manual='subheadline']"
+        )
```

Comment:
Normally `h2` is reserved for titles so that seems a bit odd to me. Could you explain the reasoning why using both?

Reply:
I added it for backwards compatibility. But that was based on the thought that older articles might also be parsed by accessing the url. When I looked though older articles I found at least a case or two where the subheadings used h2 and I thought it couldn't hurt to add that to the selectors as well